Measure unit test fixes

eringold opened the pull request:
Pull request overview
---------------------

<!--- DESCRIBE PURPOSE OF THIS PULL REQUEST -->

Fixes failing tests for: 
- ChangeBuildingLocation
- upgrade_env_roof_insulation_aedg
- EnvStormWindowsTest (measure did not exist)
- upgrade_hvac_economizer

### Pull Request Author

This pull request makes changes to (select all the apply):
 - [ ] Documentation
 - [ ] Infrastructure (includes singularity image, buildstock batch, dependencies, continuous integration tests)
 - [ ] Sampling
 - [ ] Workflow Measures
 - [x] Upgrade Measures
 - [ ] Reporting Measures
 - [ ] Postprocessing

Author pull request checklist:
<!--- Add to this list or remove from it as applicable.  This is a simple templated set of guidelines. -->
 - [ ] Tagged the pull request with the appropriate label (documentation, infrastructure, sampling, workflow measure, upgrade measure, reporting measure, postprocessing) to help categorize changes in the release notes.
 - [ ] Added tests for new measures
 - [x] Updated measure .xml(s)
 - [ ] Register values added to `comstock_column_definitions.csv`
 - [ ] Both `options_lookup.tsv` files updated
 - [ ] 10k+ test run
 - [ ] Change documentation written
 - [ ] Measure documentation written
 - [ ] ComStock documentation updated
 - [ ] Changes reflected in example `.yml` files
 - [ ] Changes reflected in `README.md` files
 - [ ] Added 'See ComStock License' language to first two lines of each code file
 - [ ] Implements corresponding measure tests and indexing path in `test/measure_tests.txt` or/and `test/resource_measure_tests.txt`
 - [x] All new and existing tests pass the CI

### Review Checklist

This will not be exhaustively relevant to every PR.
 - [x] Perform a code review on GitHub
 - [x] All related changes have been implemented: data and method additions, changes, tests
 - [x] If fixing a defect, verify by running develop branch and reproducing defect, then running PR and reproducing fix
 - [ ] Reviewed change documentation
 - [ ] Ensured code files contain License reference
 - [ ] Results differences are reasonable
 - [ ] Make sure the newly added measures has been added with tests and indexed properly
 - [ ] CI status: all tests pass

#### ComStock Licensing Language - Add to Beginning of Each Code File
```
# ComStock™, Copyright (c) 2023 Alliance for Sustainable Energy, LLC. All rights reserved.
# See top level LICENSE.txt file for license terms.
```

eringold committed:
change_building_location test fixes

eringold committed:
fix EnvRoofInsulAedgTest

eringold committed:
clean up EnvRoofInsulAedg

eringold committed:
remove EnvStormWindowsTest - measure not in comstock

eringold committed:
changebuildinglocation xml

eringold committed:
upgrade_env_roof_insul_aedg xml

eringold committed:
upgrade_env_new_aedg_windows xml

eringold committed:
fix failing economizer test - run with existing test epw

eringold committed:
update economizer measure xml

mdahlhausen reviewed (changes requested):
@eringold I'm getting some failures on the upgrade_env_new_aedg_windows measure tests. Also one failure on the hvac_economizer_test.rb.  I'm running it with OS 3.6.1 on a windows machine. in The rest of the tests are running fine.

eringold commented:
@mdahlhausen:
@ChristopherCaradonna [claimed](https://teams.microsoft.com/l/message/19:meeting_YWViMGFiY2QtMzAyYS00Zjc1LWIyZWItOTI3YWU3ZjQ4ZDI1@thread.v2/1709755669331?context=%7B%22contextType%22%3A%22chat%22%7D) upgrade_env_new_aedg_windows, I just updated the xml because I deleted the erroneously-placed EnvStormWindowsTest from that measure's test dir. 

Re: hvac_economizer_test, I see all tests passing with OS 3.7.0, using openstudio cli:
![image](https://github.com/NREL/ComStock/assets/10799524/cdd2763a-c2fc-4481-94a1-eecf28b07a19)
...
![image](https://github.com/NREL/ComStock/assets/10799524/2107648b-d38d-4fe2-89ef-af74c2823044)
Which test did you see failing? Were you running with system Ruby or with openstudio?

mdahlhausen commented:
@eringold it runs in 3.7, not 3.6.1. good to merge since we are running ComStock with OS 3.7
